app/components/network_indicator/network_indicator.js

```diff
@@ -149,7 +150,9 @@ export default class NetworkIndicator extends PureComponent {
             this.serverReachable = serverReachable;
 
             if (serverReachable) {
+                this.statusUpdates = true;
                 this.initializeWebSocket();
+                startPeriodicStatusUpdates();
```

Comment:
👍 Yes this looks good to me: "on connect if the network is reachable then start the status update polling".